src/components/Articles/CompanyProvisions.vue

```diff
@@ -145,7 +145,6 @@ export default class CompanyProvisions extends Mixins(CommonMixin) {
   @Watch('provisionsRemoved')
   private onProvisionsRemovedPropValueChanged (): void {
     if (!this.haveChanges) {
-      this.originalProvisionsRemovedValue = this.provisionsRemoved
```

Comment:
Will the originalProvisionsRemovedValue always start with false?

Reply:
Well it's a property local to the component, so it was always going to be whatever the prop value is. 

So when you resume a draft with a `provisionsRemoved` value of `true` it was setting the original value too ALSO `true`. It wasn't really a baseline "original" value. 

That being said, i'm not sure if a company could start an alteration with a provisionsRemoved value of `true`. 
I'll ask business on this one.

Reply:
So yea, this was a really good question! It may involve a little more design but i believe this is handled in the back end. 

TLDR: `hasRestrictions` is a boolean that is passed in the `businessInfo`, which dictates whether we capture this boolean value ( resolved pre existing provisions ). 

If `hasRestrictions` is true, it would show the component and indicate the pre existing provisions have not been resolved. So it is safe to assume at the component level that the original value would be false if we are rendering the component. 

After filing, i would assume the back end would resolve the `hasRestrictions`.

Reply:
My concern is about editing draft alteration where that could be selected before.

I think it makes sense to 'original' in this.originalProvisionsRemovedValue because it is original value since the component was mounted as it is updated only if there was no changes.

Reply:
Oh i see. Well in a DRAFT alteration is was broken because it was setting the original value to `true` if the user selected the checkbox, then saved / resumed a draft.

Reply:
Thats what this fix is for, to not overwrite the default value ( always `false` in this case ). So it could identify a diff between the state value of `provisionsRemoved`.

Reply:
> Oh i see. Well in a DRAFT alteration is was broken because it was setting the original value to `true` if the user selected the checkbox, then saved / resumed a draft.

In this case, I think it is correct because what we call 'original' here is there original value since the user started to work on the alteration. Not original since it was previously filed.

Reply:
Ok, after we chat I got what you're saying. You're right! Thanks!